src/recordlinker/models/algorithm.py

```diff
@@ -16,6 +17,16 @@
 LOGGER = logging.getLogger(__name__)
 
 
+@dataclasses.dataclass
+class BoundEvaluator:
+    """
+    The schema for a bound evaluator record.
```

Comment:
What is meant by "bound" evaluator? Does this refer to a constrained set of evaluator functions as above, or to "bound" as in str -> Python func?

Reply:
It just means bound to a Callable.  An `Evaluator` has two attributes, a feature (string) and a func (string).  A `BoundEvaluator` has the same two attributes, but the func is a Callable and not a string.